web/lib/firebase/notifications.ts

```diff
@@ -91,8 +72,39 @@ export function listenForNotifications(
   userId: string,
   setNotifictions: (notifications: Notification[]) => void
 ) {
-  return listenForValues<Notification>(
-    getNotificationsQuery(userId),
-    setNotifictions
+  const notifsCollection = collection(db, `/users/${userId}/notifications`)
+  const q = query(
+    notifsCollection,
+    orderBy('createdTime', 'desc'),
+    // Nobody's going through 10 pages of notifications, right?
+    limit(NOTIFICATIONS_PER_PAGE * 10)
+  )
+  return listenForValues<Notification>(q, setNotifictions)
+}
+
+export function listenForUnseenNotifications(
```

Comment:
I don't really agree with that, but if the choices are "one function that takes a flag where it can either do A or B" and "one function that does A, and one function that does B" then I prefer the latter. If the options were doing more things than just being a flag, I would feel differently.